tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test.correct

```diff
@@ -0,0 +1,6 @@
+"""
+:param t:
```

Comment:
yup i agree, what can i do to fix this? you sure about this one to be included in the first merge of DocumentationStyleBear?